server/repository/migrations/postgres/2023-03-05-222957_create_name_store_join_trigger/up.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TYPE changelog_table_name ADD VALUE 'name_store_join' AFTER 'name';
```

Comment:
the value `name_store_join` already exists, having it here will cause the server to panic!
( is in `22022-01-27T08-00_create_changelog_table/up.sql` )